readthedocs/projects/forms.py

```diff
@@ -226,10 +216,30 @@ class Meta:
             'enable_pdf_build',
             'enable_epub_build',
         )
+        fields = (
+            *per_project_settings,
+            *per_version_settings,
+        )
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        self.helper = FormHelper()
+        help_text = render_to_string(
+            'projects/project_advanced_settings_helptext.html'
+        )
+        self.helper.layout = Layout(
+            Fieldset(
+                _("Global settings"),
+                *self.Meta.per_project_settings,
+            ),
+            Fieldset(
+                _("Other settings"),
```

Comment:
I don't like this kind of division, but these are not per version settings (at least in this page)

Reply:
I liked Anthony's idea of "Default Settings" which implies they can be overwritten by the YAML, which is good. 
```suggestion
                _("Default settings"),
```